src/gui/gui.c

```diff
@@ -158,7 +158,7 @@ dt_gui_win_init_vk(dt_gui_win_t *win)
   };
   VkDescriptorPoolCreateInfo pool_info = {
     .sType         = VK_STRUCTURE_TYPE_DESCRIPTOR_POOL_CREATE_INFO,
-    .flags         = VK_DESCRIPTOR_POOL_CREATE_FREE_DESCRIPTOR_SET_BIT,
+    .flags         = VK_DESCRIPTOR_POOL_CREATE_FREE_DESCRIPTOR_SET_BIT | VK_DESCRIPTOR_POOL_CREATE_UPDATE_AFTER_BIND_BIT,
```

Comment:
why did you have to change this one? trying a few things locally with your patch, this causes validation layer complaints on my end. this appears to become a problem  when uploading individual elements for array textures, where the command buffer is used in graph-run-nodes-upload.h:84. even `vkResetCommandBuffer` will not work without this flag so i'm not sure this is a good idea. i suppose any kind of animations might trigger an error.

Reply:
Okay, I may have changed too much. This change was a consequence of this warning:

```
BestPractices-vkCreateCommandPool-command-buffer-reset(WARN / PERF): msgNum: 141128897 - Validation Performance Warning: [ BestPractices-vkCreateCommandPool-command-buffer-reset ] | MessageID = 0x86974c1 | vkCreateCommandPool(): pCreateInfo->flags has VK_COMMAND_POOL_CREATE_RESET_COMMAND_BUFFER_BIT set. Consider resetting entire pool instead.
```

I removed the `VK_COMMAND_POOL_CREATE_RESET_COMMAND_BUFFER_BIT` flag from command pool and it lead to this errors while calling `vkBeginCommandBuffer`, I poke around many flags and I fixed it calling `vkResetCommandPool` on graph.c:371 and forgot to revert this change.